lib/inspec/resources/docker_image.rb

```diff
@@ -48,6 +48,23 @@ def tag
       object_info.tags[0] if object_info.entries.size == 1
     end
 
+    def [](hashkeys)
+      hash_value = image_inspect_info
```

Comment:
Consider implementing this in terms of `dig()` - for example you could get the same result from 
```
hashkeys = hashkeys.split(".").map(&:to_sym)
value = image_inspect_info.dig(*hashkeys)
```
If one of the keys is bad, value will be nil.

Reply:
Clean 🥇